jib-core/src/main/java/com/google/cloud/tools/jib/cache/CacheStorageWriter.java

```diff
@@ -88,7 +88,13 @@ private static void moveIfDoesNotExist(Path source, Path destination) throws IOE
                     // previously-cached layer.
                     return true;
                   }
-                  Files.move(source, destination);
+                  try{
+                    Files.move(source, destination);
+                  } catch (FileSystemException e){
```

Comment:
It's quite possible that instead of here, we show this message down at line 104 (`if (!success)`) as one of the possibilities that a move/copy failed.

Reply:
Good point, done.